MegaQuests/src/main/java/com/elmer/megaquests/ItemBuilder.java

```diff
@@ -18,15 +17,19 @@ public ItemBuilder(Material material) {
 
     public ItemBuilder withDisplayName(String displayName) {
         ItemMeta itemMeta = item.getItemMeta();
-        itemMeta.setDisplayName(displayName);
-        item.setItemMeta(itemMeta);
+        if (Objects.nonNull(itemMeta)) {
+            itemMeta.setDisplayName(displayName);
+            item.setItemMeta(itemMeta);
+        }

```

Comment:
Pay attention to the IDE prompts, it fails to check for null because `itemMeta` can be null